src/components/userGroups/UserGroup.tsx

```diff
@@ -627,6 +627,14 @@ const CollapsedUserOrGroup: React.FC<CollapsedUserOrGroupProps> = ({
         setData((state) => ({ ...state, result: data }))
         updateCallback(index, data)
     }
+
+    function getToolTipContent(user) {
+        switch (user) {
+            case 'admin': return "Admin user cannot be edited"
```

Comment:
Can we keep this admin and system in the enum??

Reply:
updated 👍